soh/soh/OTRGlobals.cpp

```diff
@@ -1757,14 +1757,11 @@ extern "C" void AudioPlayer_Play(const uint8_t* buf, uint32_t len) {
     }
 }
 
-extern "C" int Controller_ShouldRumble(size_t i) {
+extern "C" int Controller_ShouldRumble(size_t slot) {
```

Comment:
this is only ever called from that one place in padmgr https://github.com/HarbourMasters/Shipwright/search?q=Controller_ShouldRumble